operations/node_management/Add_a_Standard_Rack_Node.md

```diff
@@ -2,7 +2,7 @@
 
 These procedures are intended for trained technicians and support personnel only. Always follow ESD precautions when handling this equipment.
 
--   An authentication token has been retrieved.
+- An authentication token has been retrieved.
```

Comment:
Should this be in a prerequisites section?

Reply:
Yes it should be

Reply:
I moved it to be the first step of the procedure.